docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -1,35 +1,26 @@
 # How To Upgrade Legacy Map Features
 
-A list of (case sensitive) changes to upgrade legacy TripleA Game XML properties and options
-to the most recent.
+A list of changes to upgrade legacy TripleA Game XML properties and options to the most recent.
  
-When changes are required, all occurrences of the code before the `>>` must be substituted
-with the code after the `>>`. In any code, the "\*" symbol means whatever (the change or deletion is
-to be applied to every given code having whatever character or characters in place of the "\*").
-
-## Upgrade from 1.8.0.9 to 1.9.0.0.3266
```

Comment:
If you completely remove every reference of the TripleA versions from which and to which to upgrade, I assume this can only mean this is for upgrading every map feature. Meaning that someone having a map made for TripleA 1.0 should be able to upgrade it to TripleA 2.2 by following the instruction in this document. Even in this case, I would make explicit that the starting point is TripleA 1.0. In any case, since, as far as I can tell, this document is still seemingly about 1.8.0.9 to 1.9.0.0.3266 upgrades only, I would keep such reference until it is correct or, at the very least, clarifying that this document is an incomplete "work-in-progress", making very clear that people should not currently expect it giving full information for upgrading to the latest whatever map they may have (because I can see it is certainly not able to assure that).

Reply:
"legacy map options and features" are legacy, it's not necessarily important to give a starting point. If someone finds something not in this list, then either it won't work or it's not legacy. If it does not work, hopefully we'll get a forum post about it to help.

>  I would keep such reference until it is correct or, at the very least, clarifying that this document is an incomplete "work-in-progress"

A warning that this might not be complete is perhaps not bad. I'd suggest against labelling it as a WIP, it'll be that way for forever (it is a living document).

Reply:
Rather a warning that it is incomplete (though it might be happen to be complete for some map needs).